generative_bayes_model/readme.md

```diff
@@ -0,0 +1,71 @@
+
+Usually we'd extract Re(t) from something like a SIR (susceptible-infected-recovered) model or an SEIR (which adds an exposed term), which are classic epidemiological compartment models.
+
+SIR model is in fact what Chicagoland Pandemic Response Commons used in the beginning. However, SIR or SEIR models are also just approximations of the real thing and come with quite a few assumptions baked in. The current model is simpler and makes fewer assumptions. In addition, the SIR model is described as an ODE (Ordinary Differential Equations) which causes various technical problems. Solving the ODE is quite time-intensive and while closed-form approximations exist and are faster, we found that they are quite unreliable.
+
+Instead, the newly proposed model uses a simple generative logic to explain how an initial pool of infected people spreads the disease at each time point, according to the current reproduction factor.
+
+## Objectives
+
+* Build generative models that take into account knowledge we have about the daily new incident data generation process.
+
+* Apply Bayes formula to reason backwards from observed data to unobserved hidden causes.
+
+* Model time-varying processes.
+
+* Model Infection delay (generation time).
+
+
+
+## Model Design
+
+1. Primary infection: <img src="https://render.githubusercontent.com/render/math?math=y_t%20=%20y_{t-1}%20\cdot%20R_0">
+
+2. Time-varying reproduction rate: <img src="https://render.githubusercontent.com/render/math?math=y_t%20=%20y_{t-1}%20\cdot%20R_e(t)">
+
+3. Infection delay (generation time): <img src="https://render.githubusercontent.com/render/math?math=y_t%20=%20\sum_{i=1}^{M}y_{t-i}%20R_e(t-i)%20g_i">
+
+4. Onset delay: Onset delay is defined as the delay between noticeable symptoms and reported as a positive case. To estimate onset delay distribution, we could use individual-level line list data from the [Open COVID-19 Data Working Group](https://github.com/beoutbreakprepared/nCoV2019/tree/master/latest_data ) which asked patients how long ago their symptoms started.
+
+
+## Results and Exprected Outputs
+
+* Reproduction rate prediction in Cook County:
+
+![Alt text](images/cook_county_rt.svg?raw=true "Title")
+
+* Daily confirmed cases prediction (with 15-days forecast) in Cook County:
+
+![Alt text](images/cook_county_daily.svg?raw=true "Title")
+
+## Directory layout
+        .
+        ├── model_ealuation               # Direcory to evaluate the model prediction power
+        │   ├── model_evaluation.py       # Python scripts to evaluate model prediction power
+        │   ├── images                    # Exprected output for model evaluation
+        │   └── readme.md                 # How to use model_evaluation.py
+        ├── Dockerfile                    # Dockerfile for running generative model and model evaluation
```

Comment:
`and model evaluation` => we don't run the model evaluation in the job, and we don't publish the evaluation chart on the portal, although we could if we wanted to

Reply:
you are correct, we don't publish evaluation results to portal. But users may need the docker file to build their own image to run the evaluation code, and that's why I add `Dockerfile for running generative model and model evaluation`

Reply:
i think it's not clear that the dockerfile in this directory doesn't do that

Reply:
Got it. I just removed the `and model evaluation`.